qiskit/transpiler/passes/layout/dense_layout.py

```diff
@@ -68,8 +68,14 @@ def run(self, dag):
             raise TranspilerError(
                 "A coupling_map or target with constrained qargs is necessary to run the pass."
             )
+        if self.target is not None:
```

Comment:
To be consistent with the other passes, could you make this one inline as well?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9927/commits/34a759e302867c00a465592230d6cebfaa58234e